Features/AdminComponentPreview/README.md

```diff
@@ -0,0 +1,22 @@
+# Admin Component Preview (Flynt Feature)
+
+Add a full width layout screenshot of a component to its component folder to show a preview in the Wordpress back end and front end.
+
+Components previewed in multiple scenarios:
+
+## Front end
+
+When logged in, you can overlay the screenshots with opacity in the frontend for easy pixel perfect development by clicking the 'Component Preview' button in the WordPress Admin Bar, drag the images around and move each image with the arrow keys.
+
+## Back end
+
+The preview image will be shown beside the component name in the flexible content group layout.
+
+On rollover on a component name inside the "Add Component" dropdown menu, it will show the desktop and mobile preview images.
```

Comment:
Again would avoid any passive tense:

'Rollover a component name inside the "Add Component" dropdown menu to see the desktop and mobile preview images."